site/_posts/2017-04-10-Announcing-Euwe-2.1.html.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Announcing Euwe-2.1
+author: jprause
+date: 2017-04-10 08:35:39 UTC
+comments: true
+published: true
+tags: releases announcements
+---
+
+We've just built Euwe-2.1. This release resolves the issue in Euwe-2, where dropdowns and trees don't work in the UI. There is also a manual fix found [here](http://talk.manageiq.org/t/euwe-2-broken-ui-fix/2221).
+
+We encourage you to [download](http://manageiq.org/download/) Euwe-2.1 and try it out!
```

Comment:
Since this is a minor release, I think it might be better to say something like:

```
If you already have Euwe-2 installed, you can follow the steps [here](http://talk.manageiq.org/t/euwe-2-broken-ui-fix/2221) for the manual fix.

Otherwise, we encourage you to [download](http://manageiq.org/download/) Euwe-2.1 and try it out!
```